odmantic/engine.py

```diff
@@ -291,6 +293,39 @@ def _prepare_find_pipeline(
         pipeline.extend(BaseEngine._cascade_find_pipeline(model))
         return pipeline
 
+    def _prepare_document_updates(
+        self,
+        instance: ModelType,
+        *,
+        collection_updates: Union[CollectionUpdatesType, None] = None,
+    ) -> CollectionUpdatesType:
+        """Perform an atomic save operation in the specified session"""
```

Comment:
This would be a bit clearer:
```suggestion
        """Generate update operations"""
```

Reply:
Right! Copy-paste mistake 🙈